### DISCUSSION TITLE
All fans go missing and get errors when PC wakes from sleep

### DISCUSSION THREAD

**Discussion by Lordvader81** [Rem0o/FanControl.Releases#2076] [Q&A]
When waking the PC from sleep all of my fans go missing and I get errors when restoring configuration files.
The only thing that fixes it is restarting the computer.
I have attached screenshots.

Does anybody know how to fix this problem ?
I never had this problem with my previous PC but this is a new build.

![image](https://github.com/Rem0o/FanControl.Releases/assets/120199915/b267fe49-e8ea-4178-bda8-3c3cc1a5f8b0)
![image](https://github.com/Rem0o/FanControl.Releases/assets/120199915/8a22dc80-9924-45eb-919d-304e7516b3de)

**Comment by solalla1972** [comment]
Hola.
Tengo el mismo problema en la version 172.
Mi sistema es windows 10.-
